pages/index.py

```diff
@@ -34,27 +34,42 @@ def slider_item(title, text, active=""):
             slider_item(_("Avoid Censorship"), md(_("""
 Freenet is a platform for censorship-resistant communication and publishing.
 It helps you to remain anonymous, and communicate without fear.
+
+*'Daddy, where were you when they took freedom of the press away from the Internet?'*  
+— Mike Godwin ([about Freenet](about.html))
```

Comment:
ah, yes… I guess the link title should rather be something like “the focus of Freenet”.